apps/api/src/app/environments/environments.controller.ts

```diff
@@ -20,7 +32,8 @@ export class EnvironmentsController {
     private createEnvironmentUsecase: CreateEnvironment,
     private getApiKeysUsecase: GetApiKeys,
     private getEnvironmentUsecase: GetEnvironment,
-    private getMyEnvironmentsUsecase: GetMyEnvironments
+    private getMyEnvironmentsUsecase: GetMyEnvironments,
+    private updateEncryptionUsecase: UpdateEncryption
```

Comment:
Do we have a an update environment usecase we can use for that?

Reply:
I will create widget setting update route.
/v1/environments/widget/settings